services/state/schedule/schedule.proto

```diff
@@ -109,7 +109,7 @@ message Schedule {
      * The schedule will be removed from global network state after the network reaches a consensus
      * time greater than or equal to this value.
      */
-    Timestamp calculated_expiration_time = 11;
+    int64 calculated_expiration_seconds = 11;
```

Comment:
This should be singular, it is not a duration, it is a point in time.

Reply:
Fixed.

Reply:
Fixed.